budgetportal/models.py

```diff
@@ -1595,12 +1595,20 @@ class InfrastructureProjectPart(models.Model):
     featured = models.BooleanField()
     budget_phase = models.CharField(max_length=255)
     project_slug = models.CharField(max_length=255)
-    amount = models.BigIntegerField(default=0)
+    amount_rands = models.BigIntegerField(blank=True,null=True,default=None)
     financial_year = models.CharField(max_length=4)
-    total_project_cost = models.BigIntegerField(default=0)
+    project_value_rands = models.BigIntegerField(default=0)
     provinces = models.CharField(max_length=510, default="")
     gps_code = models.CharField(max_length=255, default="")
 
+    # PPP fields
+    partnership_type = models.CharField(max_length=255, name='partnership_type', default='')
+    date_of_close = models.CharField(max_length=255, name='date_of_close', default='')
+    duration = models.CharField(max_length=255, name='duration', default='')
+    financing_structure = models.CharField(max_length=255, name='financing_structure', default='')
+    project_value_rand_million = models.CharField(max_length=255, name='project_value_rand_million', default='')
+    form_of_payment = models.CharField(max_length=255, name='form_of_payment', default='')
```

Comment:
Could you make these null=True, blank=True rather than default=''?
name=... is unneccesary, isn't it? Please remove unless it's really needed.